addons/account/tests/test_taxes_computation.py

```diff
@@ -465,6 +466,37 @@ def test_random_case_8(self):
         )
         self._assert_tests(tests)
 
+    def test_intracomm_taxes(self):
+        """ Test the base of the intracomm taxes is 100% of the base and not 0%. """
```

Comment:
I understand the test but you explain an issue in your commit message and you don't test it. In fine, you're testing the same thing, but maybe we could improve your commit message to explain what was the problem. Thank you!